timer/db.py

```diff
@@ -62,9 +66,9 @@ def get_activities(name: Optional[str]) -> Counter[str]:
             )
         results = session.exec(statement)
         for row in results:
-            activities[row.name] += row.duration_in_seconds
+            activities[row.name] += row.duration_in_minutes
```

Comment:
this is a bug actually, mypy complains about value of Counter being a float not in, will fix in separate pr